README.md

```diff
@@ -2141,6 +2142,92 @@ def reply_raw() -> manual[RawReply]:
 
 [Not yet implemented.](https://github.com/demergent-labs/azle/issues/496)
 
+### Timer APIs
+
+-   [clear timer](#clear-timer)
+-   [set timer](#set-timer)
+-   [set timer interval](#set-timer-interval)
+
+#### clear timer
+
+Examples:
+
+-   [timers](/examples/timers)
+
+```python
+from kybra import ic, TimerId, update
+
```

Comment:
There should probably an extra newline here

Reply:
I'm assuming you mean between the import statement and the `@update` decorator. if I got this wrong then let me know but I'm marking this resolved.